pkg/handlers/builders_lint.go

```diff
@@ -62,6 +63,7 @@ func LintBuildersRelease(c *gin.Context) {
 				continue
 			}
 
+			numChartsRendered += 1
```

Comment:
Style comment: Any reason for using an integer as a sentinel instead of boolean since we just want at least one?

Reply:
Makes no difference to me